spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -382,11 +380,18 @@ def when_I_visit_the_scheduling_page
   end
 
   def and_I_fill_in_the_scheduling_form
-    fill_in 'scheduled_at', with: schedule_time
+    fill_in 'schedule[date][year]', with: "2030"
+    fill_in 'schedule[date][month]', with: "04"
+    fill_in 'schedule[date][day]', with: "20"
+    fill_in 'schedule[time]', with: "10:26am"
   end
 
   def and_I_fill_in_the_scheduling_form_with_a_date_in_the_past
-    fill_in 'scheduled_at', with: '1937-04-20 10:26:51 London'
+    yesterday = Time.current - 1.day
```

Comment:
Why has the test date changed?

Reply:
No reason, felt this read a bit more clearly (the repetition of `yesterday` hitting home that it's a time in the past. Otherwise `fill_in 'schedule[date][month], with: '04'` could be perfectly applicable for dates in the past _or_ the future.

Reply:
The "1937" covers that quite well.

Reply:
It also comes in handy in a later commit, where the `yesterday` variable is used in testing that the user's inputs are retained when they hit submit and there is an error.